src/views/sign-out.njk

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.njk" %}
+
+{% set pageTitle = i18n.signOutPage.title %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form class="form" method="post">
+        <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+        {% include "includes/csrf_token.njk" %}
+
+        {{ govukInput({
+          value: props.previousUrl,
+          name: "previousPage",
+          classes: "govuk-visually-hidden"
+        }) }}
+
+        {{ govukRadios({          
+          classes: "govuk-radios--inline",
+          idPrefix: "sign_out",
+          name: "sign_out",
+          fieldset: {
+          legend: {
+            text: i18n.signOutPage.title,
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        hint: {
+          text: i18n.signOutPage.hint
+        },
+        items: [
+          {
+            value: "yes",
+            text: "Yes",
+            attributes: {
+              required: true
+            }
+          },
+          {
+            value: "no",
+            text: "No",
+            attributes: {
+              required: true
+            }
+          }
+        ]
+        }) }}
+
+        <details class="govuk-details" data-module="govuk-details" data-event-id="find-saved-information-details">
+          <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+              How do I find my saved applications?
```

Comment:
How do I find my saved applications? 
and
Sign in to your Companies House account and select Your filings to access your saved applications.

will also need translation

Reply:
Added translations.